### PR TITLE
Added Login API Handler

### LINKED ISSUE #45: Create Login API handler
**User story**
I want to be able to login to access my dashboard

**Acceptance Criteria**
User login credentials should be validated by firebase
User login credentials should be used to access obtain token

### PR DESCRIPTION
**Pull request by Pranavan10**
**Issue**
User needs to login to access database items
Resolves #45 

**Solution**
Used firebase to authenticate both email and password to return token for user-id

**Risk**

**Reviewed by**
Jonathan, Jenna

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -27,9 +29,9 @@ app.use('/users', usersRouter);
 app.use('/todo', todoRouter);
 
 // catch 404 and forward to error handler
-app.use(function (req, res, next) {
-    next(createError(404));
-});
+// app.use(function (req, res, next) {
```
Comment (Jennak21) [inline server/app.js:32]: nitpick - can this commented code be removed

Comment (Pranavan10) [reply server/app.js:32]: it shall be done

**Review by Jennak21 — APPROVED** [review]
👍

**Review by jonathan-thamrin — APPROVED** [review]
👌

**Pull request merged**